src/features/configure/pools.js

```diff
@@ -23,6 +23,30 @@ export const pools = [
     callFee: 0.5,
   },
 
+  {
+    id: 'cake-hget-bnb',
+    logo: 'cake/hget-bnb.png',
+    name: 'HGET-BNB LP',
+    token: 'HGET-BNB LP',
+    tokenDescription: 'Uses: Pancake',
+    tokenAddress: '0x66b9e1eac8a81f3752f7f3a5e95de460688a17ee',
+    tokenDecimals: 18,
+    tokenDescriptionUrl: '#',
+    earnedToken: 'mooPancakeHGET-BNB',
+    earnedTokenAddress: '0x48Cb3540CB0BB4C6dDF638209e0631Ec85c1a477',
```

Comment:
this is mooBDO, should be Vault address `0xf34629872aC0e9152C1E4204b253143e619aFf0A`

Reply:
Ooh, gotcha, thanks!